index.html

```diff
@@ -45,5 +45,269 @@ <h2 class="occupation">I am a Software Developer</h2>
             </li>
         </ul>
     </section>
+    <section class="works">
+        <h1 class="worksh">My Recent Works</h1>
+        <div class="divline"></div>
+        <section class="stories">
```

Comment:
- [ ] Kindly change the section tag to a div for each project card, this would be more semantic👍🏿